tarodb/sqlite/migrations/000007_tx_chain_fees.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE chain_txns ADD chain_fees INTEGER;
+
```

Comment:
We're still in alpha and already know that users will need to drop their DB state with the next release. So this can be added directly to the original migration file.